SPEC.md

```diff
@@ -549,6 +549,15 @@ The AC Name Type is used as the primary key for a number of fields in the schema
 The schema validator will ensure that the keys conform to these constraints.
 
 
+## AC Version Type
+
+The application container specification aims at being a forward compatible specification. Where as you are complaint to version 1.x, then your application will maintain compatibility in the 1.x major version series.
```

Comment:
The App Container specification aims to follow semantic versioning and retain forward and backwards compatibility within major versions. For example, if an implementation is compliant against version 1.0.1 of the spec, it shall remain compatible with the complete 1.x series.